### PR TITLE
Framework7 2.3.1

### PR DESCRIPTION
**Pull request by Max13**

### CODE REVIEW

File: docsets/Framework7/docset.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Framework7",
+    "package": "f7",
+    "version": "2.3.1",
+    "archive": "versions/Framework7-2.3.1.tgz",
```
Comment (Kapeli) [inline docsets/Framework7/docset.json]: This can't point to an archive inside `specific_versions`. Add it separately somewhere, even if it's a duplicate.

**Comment by Kapeli** [comment]
Failed validation:

```
Checking Framework7/versions/Framework7-2.3.1.tgz...
Empty name found for row id "329" at "Framework7.docset"
```

This means that your docSet.dsidx contains an entry without a name.

File: docsets/Framework7/docset.json (hunk 1 above)
Comment (Max13) [reply docsets/Framework7/docset.json]: Sorry again for that, it's now fixed

**Comment by Max13** [comment]
I've removed the empty line by hand from the docSet.dsidx. Apparently there is an empty <h*> in the doc...

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**